spec/02-integration/07-sdk/02-log_spec.lua

```diff
@@ -121,3 +123,339 @@ describe("PDK: kong.log", function()
     f:close()
   end)
 end)
+
+for _, strategy in helpers.each_strategy() do
+  describe("PDK: kong.log.serialize [#" .. strategy .. "]", function()
+    describe("http subsystem 1", function()
```

Comment:
Could we use more meaningful description for test suite?

Reply:
update